pkg/utils_test.go

```diff
@@ -0,0 +1,24 @@
+package pkg
+
+import "testing"
+
+func TestGetMajorMinorVersion(t *testing.T) {
+	tests := []struct {
+		version string
+		want    string
+	}{
+		{"1.2.3", "1.2"},
+		{"10.20.30", "10.20"},
+		{"0.1.2", "0.1"},
+		{"5.6", "5.6"}, // Edge case with no patch version
```

Comment:
add another edge case. you are asking for `0.4` but you have only major version, example `1`

Reply:
I'm not quite sure I understand what you're asking. Something like `{"1", "1.0"}`?

Reply:
Yes.